area.java

```diff
@@ -1,9 +1,19 @@
 import java.util.*;
 
-class Area {
+
+class Main {
 
   public static void main(String[] args) {
 
+    shapes();
+
+  }
+
+
+
+    public static void shapes(){
```

Comment:
Nice job using a `static` method! Can you think of a way to have this method have a return type other than `void`?

Reply:
so I'm trying to do return statement but I am not able to do it for example if you look at b =1 I have a variable area1 and then I'm printing that but I am not trying to do that I'm trying to return area1 and then print that in main method but when I do that it's giving me error

Reply:
It looks like from what you have right now that you have everything in the `shapes()` method. Why don't you first try splitting each up into smaller methods? Like say a `areaSquare()` method, etc. Does that make sense?

Reply:
then I would have to remove the menu option

Reply:
I don't think you would. What if, for each menu option, you created a static method?